aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -115,7 +139,9 @@ private AWSDataStorePlugin(
      * If remote synchronization is enabled, it will be performed via {@link Amplify#API}.
      *
      * @throws DataStoreException If it is not possible to access the code-generated model provider
+     * @deprecated Use {@link Builder} instead.
      */
+    @Deprecated
```

Comment:
i dont think we should deprecate them. Builder methods should provide the extra knobs when in need, but `new AWSDataStorePlugin()` constructor should still be available for basic use-case customers. It should return a default plugin (`AWSDataStorePlugin.builder().build()`) instance.

Reply:
Removed the deprecation decorator.